dockertest/subtestbase.py

```diff
@@ -246,6 +246,6 @@ def logtraceback(cls, name, exc_info, error_source, detail):
                                               exc_info[1],
                                               exc_info[2])
 
-        error_tb = "".join(error_tb).strip() + "\n"
```

Comment:
R0204:249,8: SubBase.logtraceback: Redefinition of error_tb type from list to str